db/scripts/sanitise.sql

```diff
@@ -0,0 +1,101 @@
+-- ActiveStorage
+
+DELETE FROM "active_storage_attachments";
+DELETE FROM "active_storage_blobs";
+DELETE FROM "active_storage_variant_records";
+
+-- ApplicationForm
+
+UPDATE "application_forms"
+SET
+  given_names = 'Applicant',
+  family_name = concat('Id-', id),
+  date_of_birth = '2000-01-01',
+  alternative_given_names = '',
+  alternative_family_name = '',
+  registration_number = '';
+  
+-- AssessmentSectionFailureReason
+
+UPDATE "assessment_section_failure_reasons"
+SET
+  assessor_feedback = CASE WHEN assessor_feedback IS NULL THEN NULL ELSE '[sanitised]' END; 
+
+-- AssessmentSection
+-- no update required
+
+-- Assessment 
+-- no update required
+
+-- Countries
+-- no update required
+
+-- Documents
+-- no update required
+
+-- DQTTrnRequest
+-- clear these as the scheduled jobs might run against them
+DELETE FROM "dqt_trn_requests";
+
+-- EligibilityCheck 
+-- no update required
+
+-- Features
+-- no update required
+
+-- FurtherInformationRequestItem
+UPDATE "further_information_request_items"
+SET
+  failure_reason_assessor_feedback = CASE WHEN failure_reason_assessor_feedback IS NULL THEN NULL ELSE '[sanitised]' END,
+  response = CASE WHEN response IS NULL THEN NULL ELSE '[sanitised]' END;
+
+-- FurtherInformationRequest
+UPDATE "futher_information_requests"
+SET
+  failure_assessor_note = CASE WHEN failure_assessor_note IS NULL THEN NULL ELSE '[sanitised]' END;
+
+-- Note
+UPDATE "notes"
+SET
+  text = CASE WHEN text IS NULL THEN NULL ELSE '[sanitised]' END;
+  
+-- Qaulification
+-- PII?
+-- no update required
+
+-- Region
+-- no update required
+
+-- ReminderEmail
+-- no update required
+
+-- Session
+DELETE FROM "sessions";
+
+-- Staff
+-- Keep logins?
```

Comment:
This will mean that we can all login on preprod and the assessor personas will still work if we run this over and example data setup. Wondering if we'd consider this PI and if we're happy taking this out of the production space and into preprod? It's really only staff emails that would be 'exposed' but there are hashed passwords in there too. I'm sort of leaning towards wiping them. Thoughts welcomed...

Reply:
Actually given we'll likely run this in other environments too I think we should definitely wipe them. I'll update.